dlp/snippets/src/test/java/dlp/snippets/TestBase.java

```diff
@@ -38,6 +38,8 @@ abstract class TestBase {
   protected static final String SUBSCRIPTION_ID = System.getenv("PUB_SUB_SUBSCRIPTION");
   protected static final String DATASET_ID = System.getenv("BIGQUERY_DATASET");
   protected static final String TABLE_ID = System.getenv("BIGQUERY_TABLE");
+  protected static final String DATASTORE_NAMESPACE = System.getenv("DATASTORE_NAMESPACE");
+  protected static final String DATASTORE_KIND = System.getenv("DATASTORE_KIND");
```

Comment:
@bhavika-dhanwani Please update the variable names as per the reviewed doc.
“DLP_NAMESPACE_ID” and “DLP_DATASTORE_KIND”.